backend/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "backend",
   "version": "1.0.0",
-  "main": "index.js",
+  "main": "src/index.js",
   "type": "module",
   "scripts": {
     "start": "node index.js",
```

Comment:
should be `"start": "node src/index.js"` if the index.js remains in the src folder